### ISSUE TITLE
Simulate Relay does not use basic_auth credentials

### ISSUE THREAD

**Issue by luyzdeleon** [pokt-network/pocket-core#1176]
**Describe the bug**
Currently the `/v1/client/sim` endpoint used to test validator nodes relaying capabilities ignores the `basic_auth` fields in the `chains.json`.

**To Reproduce**
1. Run a pocket node with the `simulateRelay` flag, i.e: `pocket start --simulateRelay`.
2. Configure a basic_auth scheme for your non-native chain (in the `chains.json`).
3. Attempt to submit a relay to the non-native chain you configured a `basic_auth` scheme for using the following example:

`curl -X POST --data '{"chain_url":"HTTP://<non-native node URL>:<port>","payload":{"data":"{\"jsonrpc\":\"2.0\",\"method\":\"eth_getBalance\",\"params\":[\"0xe7a24E61b2ec77d3663ec785d1110688d2A32ecc\", \"latest\"],\"id\":1}","method":"POST","path":"","headers":{}}}' <your node URL>:8081/v1/client/sim`

4. Get a 401 (unauthorized) from the non-native chain as a response:

```
{
    "code": 400,
    "message": "Expected Code 200 from Request got 401"
}
```

**Expected behavior**
Get a HTTP code 200 and the appropriate relay response.

**Comment by BenVanGithub** [comment]
Do we have reason to believe that basic auth works on non-simulated requests?
The folk lore going around at the moment is that it does.

**Comment by BenVanGithub** [comment]
Note in case anyone reading this needs it... POKT-ROKT v1.1 and up uses the basic auth creds inside the chans.json to prove connectivity from a validator to a data source.... almost a good as a simulated relay.

**Comment by andrewnguyen22** [comment]
Closed in favor of #1099
